gridt/src/app/settings/settings.page.html

```diff
@@ -0,0 +1,154 @@
+<ion-header>
+  <ion-toolbar >
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>
+      Settings
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+
+<ion-content padding>
+  <!-- Refreshes page if pulled down (mobile only)-->
+  <ion-refresher slot="fixed" (ionRefresh)="refreshPage($event)" class="ion-hide-md-up">
+    <ion-refresher-content></ion-refresher-content>
+  </ion-refresher>
+
+
+
+  <ion-grid>
+    <ion-row justify-content-center>
+      <ion-col size="12" size-sm="8" offset-sm="2" size-md="6" offset-md="3" text-center>
+        <ion-spinner *ngIf="(isDisabled$ | async )"></ion-spinner>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>Profile</ion-card-title>
+          </ion-card-header>
+
+          <ion-card-content>
+            <ion-list>
+              <ion-item class="ion-text-center" lines="none">
+                <ion-avatar class="ion-text-center"  *ngIf="(identity$ | async)">
+                  <img [src]="gravatar">
+                </ion-avatar>
+              </ion-item>
+              <ion-item class="ion-text-center" lines="none">
+                <ion-label>{{ (identity$ | async)?.username }}</ion-label>
+              </ion-item>
+              <ion-item-group>
+                <ion-item-divider>
+                  <ion-label>
+                    Bio
+                  </ion-label>
+                </ion-item-divider>
+
+                <ion-item>
+                  {{ (identity$ | async )?.bio }}
+                  <ion-button
+                    slot = "end"
+                    size = "small"
+                    [disabled] = "( isDisabled$ | async )"
+                    (click) = "changeBio()"
+                  >change</ion-button>
+                </ion-item>
+              </ion-item-group>
+            </ion-list>
+          </ion-card-content>
+        </ion-card>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>Settings</ion-card-title>
+          </ion-card-header>
+          <ion-card-content>
+            <ion-list>
+              <ion-item-group>
+                <ion-item-divider>
+                  <ion-label>
+                    Account settings
+                  </ion-label>
+                </ion-item-divider>
+
+                <ion-item type="reset">
+                  <ion-label>
+                    E-mail
```

Comment:
This label is cut off at some sizes (with my email address when the view port width is 425px), which looks a little weird. I think going vertical with the labels make more sense for responsive stuff here.